reference/twig-extensions/functions/sulu_tags.rst

```diff
@@ -0,0 +1,15 @@
+``sulu_tags``
+=============
+
+Returns all tags.
```

Comment:
All tags in the system? All tags in the current request? Is not completely clear to me (maybe if you look at the example, but I would still specify that further here).